lib/wda/xcodebuild.js

```diff
@@ -52,6 +52,7 @@ class XcodeBuild {
     this.wdaRemotePort = args.wdaRemotePort;
 
     this.updatedWDABundleId = args.updatedWDABundleId;
+    this._derivedDataPath = args.derivedDataPath;
```

Comment:
why prefix with an underscore? we don't seem to do that elsewhere in this block. what is different about this var?

Reply:
oh ok, yeah i didn't see that it was already in use elsewhere in the package. fine to keep it this way then, probably should have been more picky in that earlier code review :-)